tasks/backup_recover.yml

```diff
@@ -0,0 +1,52 @@
+---
+################################################################
+# TEMPORARY TASK TO STORE THE ROLLBACK PROCEDURE
```

Comment:
this is a temporary PLAYBOOK, not TASK... technically this playbook contains a list of tasks.